src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
Well, tabIndex=0 is the default value focusable elements like form controls in all desktop browsers.